### ISSUE TITLE
Allow background image to be defined using image macro

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor-reveal.js#19]
In some cases the background image is more than a design element, it is the primary content of a slide. This is the case when a slide is shown without text but serves an educational or informative purpose in the presentation. In this case, the image should be defined using an image macro rather than as an attribute on the section.

We should recognize slides that contain an image macro with one of the present positioning roles and promote it to the background image of the slide.

Examples:

```
[#stuck]
== !

[.cover]
image::stuck.jpg[Goose with head between bars]
```

I've identified at least two positional roles so far:

* cover
* contain

**Comment by obilodeau** [comment]
I would consider this resolved when #52 is merged. What do you think?

**Comment by mojavelinux** [comment]
For sure. We even took it a few steps further, which is awesome.
